packages/react-components/src/templates/__tests__/EventsList.test.tsx

```diff
@@ -37,3 +37,31 @@ it('renders multiple event cards', () => {
     getAllByRole('heading').map(({ textContent }) => textContent),
   ).toEqual(['FirstEvent', 'SecondEvent']);
 });
+
+it('only links to events that are not cancelled', () => {
+  const { getByRole } = render(
+    <EventsList
+      {...props([
+        {
+          ...createEventResponse({}, 0),
+          href: 'http://example.com',
+          title: 'FirstEvent',
+          groups: [],
```

Comment:
```suggestion
          groups: [],
          status: 'Confirmed',
```
who knows what the fixture default status is